### PR TITLE
Install webstorm and pycharm pro Docs

### PR DESCRIPTION
**Pull request by RealDyllon**
[SCSE-17]

### CODE REVIEW

File: docs/getting-started/webstorm-pycharm.mdx [hunk 1]
```diff
@@ -13,13 +17,100 @@ You'll want to get a free student license to avoid paying the subscription fee.
 
 Sign up for the Jetbrains Educational License [here](https://www.jetbrains.com/community/education/#students)
 
+Once you click on the link, you should be brought to this page. Under -> 'For Students and teachers', click 'Apply Now'
+
+[//]: # ([Jetbrains Apply Now]&#40;./img/jetbrains_license.png&#41;)
+
+Then fill in your details and accept the terms and conditions as follows:
```
Comment (chanbakjsd) [inline docs/getting-started/webstorm-pycharm.mdx:24]: Without the images, "as follows" is out of context.

File: docs/getting-started/webstorm-pycharm.mdx [hunk 2]
```diff
@@ -13,13 +17,100 @@ You'll want to get a free student license to avoid paying the subscription fee.
...
+
+After completing the application, you should receive instructions that are sent to your email. After clicking the link, you will need to register for a new account using the same email in the application process.
+
+[//]: # (![Email Instructions]&#40;./img/email_instructions.png&#41;)
+
+### Email Instructions
+
+[//]: # (![Email Account]&#40;./img/email_account.png&#41;)
+
+Once you have set your password and registered your account, you will be redirected to your [JetBrains Account Portal][jetbrains-account-licenses]
```
Comment (chanbakjsd) [inline docs/getting-started/webstorm-pycharm.mdx:43]:
```suggestion
Once you have set your password and registered your account, you will be redirected to your [JetBrains Account Portal][jetbrains-account-licenses].
```

File: docs/getting-started/webstorm-pycharm.mdx [hunk 3]
```diff
@@ -13,13 +17,100 @@ You'll want to get a free student license to avoid paying the subscription fee.
...
+[//]: # (    linux={<PycharmLinux />})
+
+[//]: # (/>)
+
+[//]: # (```)
+
+## Link your JetBrains License in each IDE
+
+If not already linked, open your IDEs and activate your student license by logging in with the email
+used to create the JetBrains License
```
Comment (chanbakjsd) [inline docs/getting-started/webstorm-pycharm.mdx:113]:
```suggestion
used to create the JetBrains License.
```

**Comment by RealDyllon** [comment]
@teamoteh you can use the branch [docs/SCSE-17-unfinished](https://github.com/ntuscse/docs/tree/docs/SCSE-17-unfinished) to finish this.

File: docs/getting-started/webstorm-pycharm.mdx (hunk 1 above)
Comment (RealDyllon) [reply docs/getting-started/webstorm-pycharm.mdx]: Good catch, I have removed the unnecessary wording.

**Review by chanbakjsd — APPROVED** [review]
I believe the current changes should be sufficient for now. We can create a task and a separate PR for adding images/other details if needed. Good work :)

**Pull request merged**